guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -0,0 +1,77 @@
+[id="host-substatus-overview_{context}"]
+= Host Sub-status Overview
+
+A sub-status monitors only part of a host's capabilities.
+
+Currently, {Project} ships only with *Build* and *Configuration* sub-statuses. There can be more sub-statuses depending on which plugins you add to your {Project} deployment.
+
+The *build* sub-status is relevant for managed hosts and when {Project} runs in unattended mode.
+
+The *configuration* sub-status is only relevant if {Project} uses a configuration management system like Ansible, Puppet, or Salt.
+
+To view the sub-status for a host, in the {ProjectWebUI}, navigate to *Hosts* > *All Hosts* and click the host whose full status you want to inspect. You can also view substatus information in the hover help for each host.
+
+In the *Properties* table of the host details' page, you can view both the global host status with all sub-statuses.
+
+Each sub-status can define its own set of possible values that are mapped to the three global status values.
+
+The *Build* sub-status has two possible values - *pending* and *built* that are both mapped to global *OK* value.
+
+The *Configuration* status has more possible values that map to the global status as follows:
+
+.sub-statuses that map to the global *OK* status:
+
+Active::
+During the last run, some resources were applied.
+
+Pending::
+During the last run, some resources would be applied but your configuration management integration was configured to run in `noop` mode.
+
+No changes::
+During the last run, nothing changed.
+
+No reports::
+This can be both a *Warning* or *OK* sub-status.
+This occurs when there are no reports but the host uses, for example, an associated configuration management proxy or `always_show_configuration_status` setting is set to `true`, it maps to *Warning*.
+
+.Sub-status that maps to the global *Error* status:
+
+Error::
+This indicates an error during configuration, for example, a run failed to install a package.
+
+.sub-statuses that map to the global *Warning* status:
+
+Out of sync::
+A configuration report was not received within the expected interval, based on the `outofsync_interval`.
+Reports are identified by an origin and can have different intervals based upon it.
+
+No reports::
+When there are no reports but the host uses configuration management system, for example, proxy is associated or `always_show_configuration_status` setting is set to true, it maps to *Warning*. Otherwise it is mapped to OK.
```

Comment:
This is because I have stripped it of the Puppet example. Thank you. I was wondering would it be clear. 
The problem is Satellite won't support Puppet, so how do I make this agnostic. For example, if I use a Chef example, it won't be good for orcarhino. I would love for it to make sense in a generic way.

Reply:
I'll try and grab someone to help me clarify.